WooCommerce/Classes/ViewRelated/Dashboard/Settings/Domains/DomainSettingsView.swift

```diff
@@ -156,6 +163,17 @@ struct DomainSettingsView_Previews: PreviewProvider {
                                 sitePlanResult: .success(.init(hasDomainCredit: true)))),
                                    addDomain: { _, _ in })
             }
+
+            // Loading state.
+            NavigationView {
+                DomainSettingsView(viewModel:
+                        .init(siteID: 134,
+                              stores: DomainSettingsViewStores(
+                                // There is one free domain and no other paid domains.
```

Comment:
Super nit: this comment might need to be updated 😄

Reply:
Good catch, updated in https://github.com/woocommerce/woocommerce-ios/pull/8821/commits/0d34d71d828b09e83e461f37f3d55d6c1391fa99